.circleci/test_examples.py

```diff
@@ -36,7 +36,9 @@
     'vsphere_public_key': True
 }
 
```

Comment:
need also to add the PLUGINS TO UPLOAD  (the list of the others plugins that the tests use).